float32[] state
float32[] action
float32 reward
float32[] nextState
float32[] As
bool absorbing
float32[] MDP

